packages/block-library/src/navigation/index.php

```diff
@@ -71,6 +71,18 @@ function block_core_navigation_sort_menu_items_by_parent_id( $menu_items ) {
  * Helper functions used to render the navigation block.
  */
 class WP_Navigation_Block {
```

Comment:
I think this class might be too generic? WDYT? Is there a danger it becomes a dumping ground for code with no other home?

How about we make it specific to Nav block _rendering_?
```suggestion
class WP_Navigation_Block_Renderer {
```

Reply:
Done in e5c9b18fe4